src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -327,7 +324,7 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
 
     var coordinate = new C.EPSG_4326(coordCarto.longitude, coordCarto.latitude, coordCarto.altitude);
 
-    this.scene = Scene(coordinate, viewerDiv, debugMode, gLDebug);
+    this.scene = Scene('EPSG:4978', coordinate, viewerDiv, debugMode, gLDebug);
```

Comment:
Not sure how it is related to my change. I only added a parameter that was hardcoded before. I don't really deal with axes there.